archinstall/lib/applications/application_menu.py

```diff
@@ -81,6 +103,29 @@ def _prev_print_service(self, item: MenuItem) -> str | None:
 		return None
 
 
+def select_power_management(preset: PowerManagementConfiguration | None = None) -> PowerManagementConfiguration | None:
+	group = MenuItemGroup.from_enum(PowerManagement)
+
+	if preset:
+		group.set_focus_by_value(preset.power_management)
+
+	result = SelectMenu[PowerManagement](
+		group,
+		allow_skip=True,
+		alignment=Alignment.CENTER,
+		allow_reset=True,
+		frame=FrameProperties.min(tr('Power management')),
+	).run()
+
+	match result.type_:
+		case ResultType.Skip:
+			return preset
+		case ResultType.Selection:
+			return PowerManagementConfiguration(power_management=result.get_value())
+		case ResultType.Reset:
+			raise ValueError('Unhandled result type')
```

Comment:
As you're now allowing a reset with `allow_reset=true` this needs to be handled here and return the default which is `None`